### PR TITLE
Updating profile var schema in Users.js

### PR DESCRIPTION
**Pull request by spiratla12**
So I changed var profile to account for the questions represented in https://docs.google.com/document/d/1L7y3ht1GgKJoy1E-JPwJY272SY2SmnorHY975_RKehI/edit

This includes:

**- Personal Info**

1. First Name
2. Last Name
3. Phone Number
4. Email
5. Gender
6. Age (implemented as adult or not)
7. Shirt Size
8. Emergency Contact Info

**- School**

1. School
2. School year
3. Major

**- Additional Logistics**

1. Dietary Restrictions
2. Reimbursement Required
3. If so, from where?

**- Bitcamp**

1. Amount of Hackathons attended before
2. Why Bitcamp
3. What do you want to build

**- Professional**

1. What type of work opportunities are you interested in?

**- Legal**

1. Code of Conduct

### CODE REVIEW

File: app/server/models/User.js [hunk 1]
```diff
@@ -6,97 +6,129 @@ var mongoose   = require('mongoose'),
...
     min: 1,
     max: 150,
   },
 
-  graduationYear: {
+  schoolYear: {
     type: String,
     enum: {
-      values: '2016 2017 2018 2019'.split(' '),
+      values: 'High-School Freshman Sophomore Junior Senior'.split(' '),
```
Comment (nyaculak) [inline app/server/models/User.js]: This should have graduate too, I think

File: app/server/models/User.js [hunk 2]
```diff
@@ -6,97 +6,129 @@ var mongoose   = require('mongoose'),
...
-    enum : {
-      values: 'M F O N'.split(' ')
+    enum: {
+      values: '0 1-5 6-10 11-15 16-20 20+'.split(' '),
     }
   },
+  whyBitcamp: {
+    type: String,
+    min: 0,
+    max: 250
```
Comment (nyaculak) [inline app/server/models/User.js]: I think these are character counts, not word counts. We probably want to make it bigger

Comment (spiratla12) [reply app/server/models/User.js]: done. upped to 2500

File: app/server/models/User.js (hunk 1 above)
Comment (spiratla12) [reply app/server/models/User.js]: added

**Comment by jsjqian** [comment]
This looks good, but we still have some other fields that need to be added. They're listed here: https://docs.google.com/document/d/1L7y3ht1GgKJoy1E-JPwJY272SY2SmnorHY975_RKehI/edit. Could you please add those fields as well? There is also a need to update the actual application to reflect these changes. Let me know if that's something you can also do. Great work so far!

**Pull request merged**